service-api/app/src/App/src/Service/Lpa/ValidateOlderLpaRequirements.php

```diff
@@ -23,49 +26,49 @@ class ValidateOlderLpaRequirements
 
     private DateTimeImmutable $earliestDate;
     private LoggerInterface $logger;
+    private FeatureEnabled $featureEnabled;
 
     /**
      * ValidateOlderLpaRequirements constructor.
      *
      * @param LoggerInterface $logger
      * @codeCoverageIgnore
      */
-    public function __construct(LoggerInterface $logger)
+    public function __construct(LoggerInterface $logger, FeatureEnabled $featureEnabled)
     {
         $this->logger = $logger;
         $this->earliestDate = new DateTimeImmutable(self::EARLIEST_REG_DATE);
+        $this->featureEnabled = $featureEnabled;
     }
 
     /**
      * @param array $lpa An LPA data structure
-     *
-     * @return bool The LPA has valid requirements
      * @throws Exception Thrown when unable to parse LPA registration date as a date
      */
-    public function __invoke(array $lpa): bool
+    public function __invoke(array $lpa): void
     {
-        if ($lpa['status'] !== self::NECESSARY_STATUS) {
+        if (!($lpa['status'] === self::NECESSARY_STATUS)) {
             $this->logger->notice(
                 'User entered LPA {uId} does not have the required status',
                 [
                     'event_code' => EventCodes::OLDER_LPA_INVALID_STATUS,
                     'uId' => $lpa['uId'],
                 ]
             );
-            return false;
+            throw new NotFoundException('LPA status invalid');
         }
 
-        if (new DateTimeImmutable($lpa['registrationDate']) < $this->earliestDate) {
-            $this->logger->notice(
-                'User entered LPA {uId} has a registration date before 1 September 2019',
-                [
-                    'event_code' => EventCodes::OLDER_LPA_TOO_OLD,
-                    'uId' => $lpa['uId'],
-                ]
-            );
-            return false;
+        if (!(($this->featureEnabled)('allow_older_lpas'))) {
```

Comment:
```suggestion
        if (!($this->featureEnabled)('allow_older_lpas')) {
```
Extra set of brackets